src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -3044,6 +3044,131 @@ Many use cases can be fulfilled using this API. For instance:
   can scale up and down these resources when receiving the corresponding signals from users (via the Workflow Execution portal),
   or from third-party systems (via the Signals REST endpoint).
 
+== Third party services API
+
+Some ProActive services offers java Clients ready to be used in ProActive's workflows.
+
+=== ProActive Service Automation (PSA) API
+
+The ProActive Service Automation API, or ProActive Service Automation Client, is a way by which a workflow can communicate with ProActive's Service Automation service
```

Comment:
ProActive's Service Automation service => the Service Automation module.
(Also link to the Service Automation documentation)

using java code to send HTTP requests. => remove this (it is not only limited to Java)